arm-recoveryservicesbackup/2016-06-01/swagger/recoveryservicesbackup.json

```diff
@@ -0,0 +1,1548 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2016-06-01",
+    "title": "RecoveryServicesBackupClient",
+    "x-ms-code-generation-settings": {
+      "internalConstructors": true
+    }
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json",
+    "text/json",
+    "application/octet-stream"
+  ],
+  "produces": [
+    "application/json",
+    "text/json",
+    "application/octet-stream"
+  ],
+  "paths": {
+    "/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{resourceNamespace}/vaults/{resourceName}/backupFabrics/{fabricName}/protectionContainers/{containerName}": {
+      "get": {
+        "tags": [
+          "ProtectionContainers"
+        ],
+        "description": "Gets details of the specific container registered to your Recovery Services Vault.",
+        "operationId": "ProtectionContainers_BMSGetContainer",
+        "consumes": [],
+        "produces": [
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "name": "resourceName",
```

Comment:
No, you are still not getting it. 
We are saying just define resourceGroupName and resourceName once in the global parameters section and apply the `"x-ms-parameter-location": "method"` extension on them. In that way they will not be properties on the client.

By defining them once, you can reference them at multiple places in the spec there by avoiding duplication.